modules/gapi/test/s11n/gapi_s11n_tests.cpp

```diff
@@ -34,6 +34,28 @@ namespace detail {
 } // namespace gapi
 } // namespace cv
 
+
+namespace cv {
+namespace detail {
+template<> struct CompileArgTag<MyCustomType>
+{
+    static const char* tag()
+    {
```

Comment:
This file has other indentation style, please follow.

Reply:
Reworked.